presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -816,4 +817,16 @@ public FeaturesConfig setMaxGroupingSets(int maxGroupingSets)
         this.maxGroupingSets = maxGroupingSets;
         return this;
     }
+
+    public int getMaxStageCount()
+    {
+        return maxStageCount;
+    }
+
+    @Config("max-stage-count")
+    public FeaturesConfig setMaxStageCount(int maxStageCount)
```

Comment:
There’s probably a better place for this config. The name should also be prefixed with “query.” or something. Take a look at other configs.

Reply:
See QueryManagerConfig

Reply:
Also please specify a min/max.